src/pages/Donate/DonateSupportUs.vue

```diff
@@ -45,8 +45,16 @@ const pageQuery = gql`query donateContent {
 
 export default {
 	metaInfo() {
-		return {
-			title: `${this.pageTitle}`,
+		return 	{
+			title: 'Donate to Kiva and create real change',
```

Comment:
Why not just set this in Contentful seeing as it's already wired up there?

Reply:
@mcstover In my understanding this is the way it has been done in several tickets, I'm just following some previous solutions.

Reply:
I think most of the other tickets do not have a corresponding Contentful page like these 2 do.

It would be more future proof to have this title set properly in Contentful but I'm ok in the short term to move it here.

@emuvente your call.